subsys/spm/spm.c

```diff
@@ -334,6 +334,8 @@ static void spm_config_peripherals(void)
 		PERIPH("NRF_TIMER1", NRF_TIMER1_S, CONFIG_SPM_NRF_TIMER1_NS),
 		PERIPH("NRF_TIMER2", NRF_TIMER2_S, CONFIG_SPM_NRF_TIMER2_NS),
 		PERIPH("NRF_SAADC", NRF_SAADC_S, CONFIG_SPM_NRF_SAADC_NS),
+		PERIPH("NRF_PWM0", NRF_PWM0_S, CONFIG_SPM_NRF_PWM0_NS),
```

Comment:
NRF_PWMx should be defined, AFAICS.

Reply:
Looks like I need the `_S` for peripherals that are not set to `status = "ok"` in DT. Only `PWM0` is enabled for the board, so I'll have to keep `_S` for the remaining instances. I'll remove from the timers, TWI and SPI.

UARTE2 is disabled because it's reserved for MCU boot, according to @hakonfam .